lib/exceptions.js

```diff
@@ -0,0 +1,36 @@
+var records = require('./exceptions.json');
+
+function compareValue(data, ref) {
+    return (ref === undefined) || (data === ref);
+}
+
+function findSet(shortname) {
+    var count = 0;
+    function _findSet(name) {
+        if (count > 10)
+           return undefined;
+        var set = records[name];
+        count++;
+        if (typeof set === "String")
+            return _findSet(set);
+        return set;
+    }
+    return _findSet(shortname);
+}
```

Comment:
See https://github.com/w3c/specberus/pull/212
Here is an example of valid syntax:
```javascript
 "hr-time-2": [
    {
      "key": "error",
      "name": "validation.css"
    },
    {
      "key": "error",
      "extra":
      {
        "message": "Attribute for not allowed on element a"
      },
      "name": "validation.html"
    }
  ]
}
```